app/models/review.rb

```diff
@@ -1,9 +1,16 @@
 class Review < ActiveRecord::Base
   belongs_to :user
-  #belongs_to :sitcom
+  belongs_to :sitcom
 
   validates :rating, presence: true,
   numericality: { only_integer: true, greater_than: 0, less_than: 6 }
   validates :user, presence: true
-  #validates :sitcom, presence: true
+  validates :sitcom, presence: true
+  validates_uniqueness_of :user_id, scope: [:sitcom_id], message: "You have already reviewed this show."
```

Comment:
Line is too long. [104/80]<br>Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.